### ISSUE TITLE
Heterogeneous graph should specify triplet type in metadata.json

### ISSUE THREAD

**Issue by xingjian-zhang** [Graph-Learning-Benchmarks/gli#22]
Currently, we store the OGB link prediction data as a heterogeneous graph that has two edge groups
1. `PositiveEdge`
2. `NegativeEdge`

but single node group
1. `Node`

However, DGL requires different edge groups to have different relation triplets. So we need to either **specify the triplet in metadata.json for each edge group** or **simply label pos/neg as features**.
```python
import dgl
import torch as th
# Create a heterograph with 3 node types and 3 edges types.
graph_data = {
   ('drug', 'interacts', 'drug'): (th.tensor([0, 1]), th.tensor([1, 2])),
   ('drug', 'interacts', 'gene'): (th.tensor([0, 1]), th.tensor([2, 3])),
   ('drug', 'treats', 'disease'): (th.tensor([1]), th.tensor([2]))
}
g = dgl.heterograph(graph_data)
```
